include/deal.II/base/function_unit_to_real.h

```diff
@@ -0,0 +1,82 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 - 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+#ifndef dealii_function_unit_to_real_h
+#define dealii_function_unit_to_real_h
+
+
+#include <deal.II/base/function.h>
+#include <deal.II/base/point.h>
+#include <deal.II/base/smartpointer.h>
+#include <deal.II/base/tensor.h>
+
+#include <deal.II/fe/fe_nothing.h>
+#include <deal.II/fe/mapping_q1.h>
+
+#include <deal.II/grid/tria.h>
+
+
+DEAL_II_NAMESPACE_OPEN
+namespace Functions
+{
+  /**
+   * Wrapper class for transforming a function from real to unit space.
+   *
+   * @ingroup functions
+   */
+  template <int dim>
+  class FunctionUnitToReal : public Function<dim>
+  {
+  public:
+    FunctionUnitToReal(const Function<dim> &function);
```

Comment:
What is the role of this function? I think it would be useful to provide a formula that explains what this class actually does :-)